README.md

```diff
@@ -54,6 +54,10 @@ GitLab LDAP configuration; if `gitlab_ldap_enabled` is `true`, the rest of the c
 
 Gitlab timezone.
 
+    gitlab_validate_certs: yes
```

Comment:
Can you call this `gitlab_download_validate_certs` instead, just to make it a little more clear that this is validating certs only during the download of GitLab's install scripts?

The way it's named right now, it sounds like it's a GitLab setting (which it's not).